python_examples/examples/tv_example.py

```diff
@@ -0,0 +1,47 @@
+from sinric import Sinric
+import logging
+
+apiKey = ''
+
+
+def setPowerState(did, value):
+    print(did, value)
```

Comment:
Perhaps instead of `print` statements, use `logger.info(f"{did} - {value}")`

Reply:
I just used print because this code is for Rapsberry Pi mainly. So user can edit functions for using gpio pins.